api/src/client/AirtableUserTestingClient.test.ts

```diff
@@ -32,7 +33,7 @@ describe("AirtableUserTestingClient", () => {
         baseUrl: "some-base-url",
         usersTableName: "some-users-table",
       },
-      logger
+      DEBUG ? logger : DummyLogWriter
```

Comment:
What if we centralized this line of code in setupTest.js (define and invoke the set globally). we could mock the implementation and return Logger or dummyLogWriter based on the env variable. The though it not fully thought through. If it can be done, we wouldn't need to do this everytime, we'd need to simply import the module, and the mocking is done in setupTests.